src/GraphQL.Tests/Serialization/SystemTextJson/InputsConverterTests.cs

```diff
@@ -0,0 +1,302 @@
+using System.Collections.Generic;
+using System.Text.Encodings.Web;
+using System.Text.Json;
+using GraphQL.SystemTextJson;
+using Shouldly;
+using Xunit;
+
+namespace GraphQL.Tests.Serialization.SystemTextJson
+{
+    public class InputsConverterTests
```

Comment:
Another duplicate of the `ObjectDictionaryConverter` tests.  So we can just delete the `ObjectDictionaryConverterTests` when we delete the `ObjectDictionaryConverter` from 5.0.